cms/djangoapps/contentstore/toggles.py

```diff
@@ -219,6 +219,24 @@ def use_new_custom_pages(course_key):
     """
     return ENABLE_NEW_STUDIO_CUSTOM_PAGES.is_enabled(course_key)
 
+# .. toggle_name: contentstore.new_studio_mfe.use_markdown_editor
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: This flag enables the use of the Markdown editor when creating or editing problems.
```

Comment:
Since markdown editing is always enabled for the legacy problem editor, please clarify that this enables markdown specifically in the new Authoring MFE editor